### DISCUSSION TITLE
Support mod key code

### DISCUSSION THREAD

**Discussion by maclockard** [xyflow/xyflow#2658] [New Features]
### Please describe the feature that you want to propose

Several hot key libraries support the key code `mod` to help keep keyboard shortcuts consistent between platforms. This key code works as `meta` on Mac and `ctrl` on Windows.

This helps prevent people from mistakenly using `meta` not realizing its the windows key on Windows.

Examples of other popular libraries using this keycode:
* [mousetrap](https://github.com/ccampbell/mousetrap/blob/master/mousetrap.js#L135)
* [blueprint](https://github.com/palantir/blueprint/blob/develop/packages/core/src/components/hotkeys/hotkeyParser.ts#L124)

**Comment by moklick** [comment]
I converted this into a discussion, because I wanted to cleanup the issues a bit. 

I like the idea but I also want to keep the key handler features as low level as possible. I am not sure if we are going to add this.
